package.json

```diff
@@ -16,33 +18,34 @@
     "example": "examples"
   },
   "devDependencies": {
-    "@types/mocha": "^2.2.46",
-    "@types/sinon": "^4.1.3",
+    "@babel/cli": "^7.12.10",
+    "@babel/core": "^7.12.10",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-typescript": "^7.12.7",
+    "@babel/register": "^7.12.10",
+    "@types/jest": "^26.0.20",
     "@types/ws": "^3.2.1",
-    "babel-core": "^6.26.0",
-    "babel-loader": "^7.1.2",
-    "babel-preset-env": "^1.6.1",
-    "husky": "^2.4.0",
-    "mocha": "^4.0.1",
-    "should": "^13.1.3",
-    "sinon": "^4.1.5",
-    "ts-loader": "^5.2.2",
-    "ts-node": "^3.3.0",
+    "husky": "^4.3.8",
+    "jest": "^26.6.3",
+    "npm-run-all": "^4.1.5",
     "tslint": "^5.6.0",
-    "typescript": "^3.1.2",
-    "typescript-formatter": "^7.0.0",
-    "webpack": "^4.23.1",
-    "webpack-cli": "^3.1.2",
+    "typescript": "^4.1.3",
+    "typescript-formatter": "^7.2.2"
+  },
+  "dependencies": {
     "ws": "^3.3.2"
```

Comment:
Move back to devDependencies since it's only used for examples (but then examples should probably have their own `package.json`)